### DISCUSSION TITLE
Shutdown cluster on Azure Cloud?

### DISCUSSION THREAD

**Discussion by Stefan-Kroscen** [ravendb/ravendb#18614] [General]
Hi,
I have a one node dev cluster on the azure cloud.  I selected hourly billing.  Is there a way to shutdown the cluster when not in use? I am not seeing it.

Thanks!

**Comment by ayende** [comment]
No, you cannot do that, I'm afraid.
The issue is that the instances are actually alive, even if not in use. We have some plans on serverless model, which should make it easier, but that is later in the year.

The underlying issue is that provisioning the backend work on Azure is typically ~3 - 5 *minutes*, which is not an acceptable scenario for us.

**Comment by Stefan-Kroscen** [comment]
ok, thanks for clarifying that.


________________________________
